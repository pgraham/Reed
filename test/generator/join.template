This is a sample template that contains a join substitution.
This line contains a join with ',' as the glue: ${join:joined:,}
This line contains a join with ' ' as the glue: ${join:joined: }
