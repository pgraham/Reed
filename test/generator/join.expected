This is a sample template that contains a join substitution.
This line contains a join with ',' as the glue: val1,val2,val3
This line contains a join with ' ' as the glue: val1 val2 val3
